spec/migrator/alter_table_statement_spec.cr

```diff
@@ -48,6 +48,22 @@ describe Avram::Migrator::AlterTableStatement do
     built.statements[6].should eq "ALTER TABLE users ALTER COLUMN updated_at SET NOT NULL;"
   end
 
+  it "does not build statements if nothing is altered" do
+    built = Avram::Migrator::AlterTableStatement.new(:users).build { }
+    built.statements.size.should eq 0
+  end
+
+  it "can change column types" do
+    built = Avram::Migrator::AlterTableStatement.new(:users).build do
+      change_type id : Int64
+      change_type age : Int32
```

Comment:
Decided to just make it part of `alter` since we do that for indexes and `fill_existing_with`